docs/docs/contribute/software/dummy-code-example.go

```diff
@@ -0,0 +1,45 @@
+package dummy
```

Comment:
please update the examples with the new code, make sure t add comments on the expected outputs

Reply:
Alright